adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,133 @@ endif::[]
+** PXC (Percona XtraDB Cluster) replaces `cf-mysql` for database management -- please read the Known Issues section for this version on deployment and upgrade changes
+** Ability to set `config.HA_strict=false` in combination with `config.HA=true` to allow lowering the sizing count for a role below what is required for HA
+** UAA can now be embedded in the `cf` namespace -- please read the Known Issues section for this version on deployment limitations
+* The Stratos UI has been updated to version 2.5.1:
+** Tech preview for helm feature
+** Add custom welcome message on endpoints page
+** Add support for connecting {caasp} V4 endpoints
+** Refinements to the Autoscaler UI
+** For a full list of features and fixes see https://github.com/SUSE/stratos/releases/tag/2.5.1.
+
+For information about deploying and administering {product}, see the product manuals at
+{doc-url}.
+
+[id='sec.1_5.feature']
+==== Features and Fixes
+* `cf-deployment` has been updated to version 9.5.0.
+* Eirini updated to 0.0.14
+* Removed the `cluster-admin` role binding for the eirini service account
+* Switched over to `PXC` from `cf-mysql` for database management
+* Includes these {cf} component versions:
+** cf-deployment: 9.5
+** cflinuxfs3: 0.118.0
+** eirini: 0.0.14
+** groot-btrfs: 1.0.5
+** scf-helper: 1.0.3
+* Buildpacks:
+** binary-buildpack: 1.0.33
+** dotnet-core-buildpack: 2.2.13
+** go-buildpack: 1.8.42
+** java-buildpack: 4.20.0 
+** nginx-buildpack: 1.0.15
+** nodejs-buildpack: 1.6.53
+** php-buildpack: 4.3.80
+** python-buildpack: 1.6.36
+** ruby-buildpack: 1.7.42
+** staticfile-buildpack: 1.4.43
+
+[id='sec.1_5.issue']
+==== Known Issues
+
+IMPORTANT: In order to deploy this version in HA, or upgrade from {cap} 1.4.1 from HA, you will need to scale down the `mysql` role to 1 in order to migrate from cf-mysql to PXC. This is based on upstream instructions but based on what we've seen with other components that rely on the database, such as Credhub, scaling all database roles into single availability helps with a stable migration and deployment.
+
+Steps for a fresh install of CAP 1.5 HA: 
+
+[arabic]
+.. Install HA UAA but scale the `mysql` role to 1 as a transition step. (For these commands, we will assume that `scf-uaa` and `scf-cf` are the chart names.)
++
+[source,bash]
+----
+helm install --name scf-uaa --namespace uaa suse/uaa -f <values.yaml> --set config.HA=true \ 
+--set config.HA_strict=false --set sizing.mysql.count=1
+
+----
++
+.. Set the value of the `secrets.UAA_CA_CERT` to pass your `uaa` secret and certificate to `scf` as per Section 5.10 of the {CAP} Deployment Guide: https://www.suse.com/documentation/cloud-application-platform-1/singlehtml/book-cap-guides/book-cap-guides.html#sec-cap-install-scf-prod
++
+[source,bash]
+----
+helm install --name scf-cf --namespace scf suse/cf -f <values.yaml> --set config.HA=true \ 
+--set config.HA_strict=false --set sizing.mysql.count=1 --set "secrets.UAA_CA_CERT=${CA_CERT}"
+----
```

Comment:
Should be:
```
helm install --name susecf-scf --namespace scf suse/cf -f <values.yaml> --set config.HA=true \ 
--set config.HA_strict=false --set sizing.mysql.count=1 --set "secrets.UAA_CA_CERT=${CA_CERT}"
```

Reply:
Done. From what I see, all references to `--name` are updated.